client-app/src/examples/contactMobile/ContactsPageModel.ts

```diff
@@ -0,0 +1,120 @@
+import {HoistModel, managed, persist, XH} from '@xh/hoist/core';
+import {bindable, makeObservable, observable, action, runInAction} from '@xh/hoist/mobx';
+import {GridModel} from '@xh/hoist/cmp/grid';
+import {nameCol, locationCol} from '../../core/columns';
+import {uniq} from 'lodash';
+
+// Shared from the desktop version
+import {PERSIST_APP} from '../contact/AppModel';
+
+import ContactDetailsModel from './details/ContactDetailsModel';
+
+export default class ContactsPageModel extends HoistModel {
+    override persistWith = PERSIST_APP;
+
+    @observable.ref tagList: string[] = [];
+
+    @bindable @persist displayMode: 'grid' | 'tiles' = 'tiles';
+
+    @managed gridModel: GridModel;
+    @managed contactDetailsModel: ContactDetailsModel;
+
+    get selectedRecord() {
+        return this.gridModel.selectedRecord;
+    }
+
+    get records() {
+        return this.gridModel.store.records;
+    }
+
+    constructor() {
+        super();
+        makeObservable(this);
+
+        const gridModel = (this.gridModel = this.createGridModel());
+        this.contactDetailsModel = new ContactDetailsModel(this);
+
+        this.addReaction({
+            track: () => gridModel.selectedRecord,
+            run: rec => this.contactDetailsModel.setCurrentRecord(rec)
+        });
+    }
+
+    toggleFavorite(record) {
+        XH.contactService.toggleFavorite(record.id);
+        this.gridModel.store.modifyRecords({id: record.id, isFavorite: !record.data.isFavorite});
+    }
+
+    clearCurrentSelection() {
+        this.gridModel.clearSelection();
+    }
+
+    @action
+    setDisplayMode(value: 'grid' | 'tiles') {
+        this.displayMode = value;
+
+        XH.setPref(PERSIST_APP.prefKey, {
```

Comment:
See https://github.com/xh/toolbox/pull/752/files#r1994240772. This was persisting correctly alone, but if I include the above fix to user favorites and NOT this, then saving a favorite will clobber the save of mode.